src/Media_Command.php

```diff
@@ -446,24 +447,28 @@ public function image_size( $args, $assoc_args ) {
 				'width'     => intval( get_option( 'large_size_w' ) ),
 				'height'    => intval( get_option( 'large_size_h' ) ),
 				'crop'      => false !== get_option( 'large_crop' ) ? 'hard' : 'soft',
+				'ratio'			=> false !== get_option( 'large_crop' ) ? $this->get_ratio( intval( get_option( 'large_size_w' ) ), intval( get_option( 'large_size_h' ) ) ) : 'N/A',
```

Comment:
As we're using the string `'N/A'` multiple times here, you should put it into a constant and refer to the constant instead.

This way, if we want to change the value to display for these cases, we can do it consistently and without running the risk of missing one or more instances.

Reply:
> you should put it into a constant and refer to the constant instead.

Variable — constants are globally scoped.